service/src/main/kotlin/fi/kangasala/evaka/invoice/service/ProEInvoiceGenerator.kt

```diff
@@ -188,6 +188,12 @@ class ProEInvoiceGenerator(private val invoiceChecker: InvoiceChecker, val finan
         return invoiceData
     }
 
+    private fun getCreditAccounting(): String {
+        val tili = "323011"
+        val costCenter = "1913021"
+        return "$tili$costCenter"
```

Comment:
I think this should be:
```
        val toiminto = "1913021"
        return "$tili$kustannuspaikka$toiminto"
```
where `$kustannuspaikka` = `InvoiceRowDetailed#costCenter`.